front/src/app/todo-list/todo-list.component.ts

```diff
@@ -0,0 +1,43 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { ApiService, Lists } from '../shared/services/Api.service';
+import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-todo-list',
+  templateUrl: './todo-list.component.html',
+  styleUrl: './todo-list.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TodoListComponent implements OnInit{
+
+  allTaskLists: Lists;
+
+  radioBtnGroup: FormGroup = new FormGroup ({
+    activeTaskList: new FormControl()
+  })
+
+  constructor(public apiService: ApiService, private changeDetectorRef: ChangeDetectorRef, public formBuilder: FormBuilder, private route: ActivatedRoute) {
```

Comment:
set correct order of services in constructor - `public` firs, `private` second.

Review these services usage and make them `private` unless they are used in `.html`